File: Actividad 1/ejerciciorn4.py
class ejerciciorn4: #Creamos la clase con el nombre "ejerciciorn4"

    edJuan = 9 #Usamos 9 para hacer la prueba de escritorio

    #Calculamos las edades
    edAlber = int(2*(edJuan/3))
    edAna = int(4*(edJuan/3))
    edMama = edJuan + edAlber + edAna

    #Imprimimos por consola las edades
    print("Las edades son:")
    print("Alberto: " + str(edAlber))
    print("Juan: " + str(edJuan))
    print("Ana: " + str(edAna))
    print("Mamá: " + str(edMama))

File: Actividad 1/ejerciciorn4.java
package actividad1;

public class ejerciciorn4 {

    public static void main(String[] args) {
        int edJuan = 9;
        int edAlber, edAna, edMama;

        edAlber = 2*(edJuan/3);
        edAna = 4*(edJuan/3);
        edMama = edJuan + edAlber + edAna;

        System.out.println("Las edades son:");
        System.out.println("Alberto: " + edAlber);
        System.out.println("Juan: " + edJuan);
        System.out.println("Ana: " + edAna);
        System.out.println("Mamá: " + edMama);
    }
}
